manifests/base/jupyter-rocm-minimal-notebook-imagestream.yaml

```
@@ -0,0 +1,30 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/blob/main/rocm"
+    opendatahub.io/notebook-image-name: "ROCm"
+    opendatahub.io/notebook-image-desc: "Jupyter ROCm notebook image for ODH notebooks."
+    opendatahub.io/notebook-image-order: "10"
```

This is an increasing number, the pattern that we follow here is the following:
jupyter-minimal= 10
jupyter-datascience=20
CUDA=30
jupyter-pytorch=40
jupyter-tensorflow=50
jupyter-trusty=60
habana=70
code-server=80
So the rest should have 90/100/110
```suggestion
    opendatahub.io/notebook-image-order: "90"
```